core/lib/zksync_core/src/api_server/execution_sandbox/error.rs

```diff
@@ -61,6 +61,10 @@ impl From<Halt> for SandboxExecutionError {
             Halt::FailedToAppendTransactionToL2Block(reason) => {
                 SandboxExecutionError::Revert(reason, vec![])
             }
+            Halt::TracerCustom(reason) => SandboxExecutionError::Revert(reason, vec![]),
+            Halt::ValidationOutOfGas => Self::UnexpectedVMBehavior(
```

Comment:
`Self::UnexpectedVMBehavior` may not be appropriate here, since it would say that the VM should've never returned such error & that the user should contact the team

Reply:
are you suggesting Revert?